test/end-to-end/config.json

```diff
@@ -1,13 +1,4 @@
 {
-    "root": "http://localhost:3000/",
```

Comment:
wdio.conf.js already has the same settings as this one, this file should be removed.